src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -165,9 +165,6 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 			this.createTabs(tempWidgets);
 		}
 
-		this.showToolbar = true;
-		this.createToolbar(this.toolbarContainer.nativeElement, this.homeTabId);
```

Comment:
removing this doesn't break anything?

Reply:
forgot to add a comment here, yes, I was testing it and noticed this is not needed, the logic of hide/show+create toolbar will be handled by the tab change event handler when initialized.